packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SVGComponents/SVGImage/SVGImage.js

```diff
@@ -33,19 +33,30 @@ export default function SVGImage ({
     document.body.insertAdjacentHTML('afterbegin', INVERT)
   }
 
-  const SubjectImage = move ? DraggableImage : 'image'
-
-  return (
-    <SubjectImage
-      aria-label={`Subject ${subjectID}`}
-      dragMove={move ? onDrag : () => null}
-      height={naturalHeight}
-      role='img'
-      filter={invert ? 'url("#svg-invert-filter")' : undefined}
-      width={naturalWidth}
-      href={src}
-    />
-  )
+  if (move) {
+    return (
+      <DraggableImage
+        aria-label={`Subject ${subjectID}`}
+        dragMove={move ? onDrag : () => null}
+        height={naturalHeight}
+        role='img'
+        filter={invert ? 'url("#svg-invert-filter")' : undefined}
+        width={naturalWidth}
+        href={src}
+      />
+    )
+  } else {
```

Comment:
Do we have a team preference between using `else` or just returning something? I'm happy to change it however preferred.

Reply:
For clarity, what do you mean by just returning something? I think in this situation I would do the following. Your if else solution works, too.
```
return (
  {move ? (
    <DraggableImage />
  ) : (
    <image etc />
  )
)
```

Reply:
If you change the `dragMove` attribute in the original code, you don’t need an if block at all. `dragMove={ move ? onDrag : undefined}`. 

That way, you aren’t writing all the other component props twice. Of course, you could also define the component props once, outside the if statement.

Reply:
@goplayoutside3 I think Mark’s asking whether we prefer a default return for components. 

```js
if (condition === true) {
  return conditionalValue
}
return defaultValue
```

I don’t think I’ve got a strong opinion about this. However, statements have side effects in JavaScript. Expressions don’t have side effects so I’d lean towards using a ternary expression for `dragMove`.

Reply:
@goplayoutside3 - @eatyourgreens is correct, I was referring to if we have a team preference for a default return or using `else`.

@eatyourgreens:
- thank you for noting statements have side effects while expressions don't, not sure if I've forgotten or ever knew, but that's interesting and good to know!
- if I change the original code to `dragMove={ move ? onDrag : undefined}` I still get the noted console warning (only in dev environment)

I've made an update, combining the great suggestions, thank you both!